vaadin-form-layout-flow-parent/vaadin-form-layout-flow-integration-tests/src/test/java/com/vaadin/flow/component/formlayout/tests/FormLayoutIT.java

```diff
@@ -23,17 +23,23 @@
 import org.openqa.selenium.Dimension;
 import org.openqa.selenium.WebElement;
 
-import com.vaadin.flow.component.formlayout.demo.FormLayoutView;
-import com.vaadin.tests.ComponentDemoTest;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.testbench.TestBenchTestCase;
+import com.vaadin.tests.AbstractComponentIT;
 
 /**
  * Integration tests for the {@link FormLayoutView}.
  */
-public class FormLayoutIT extends ComponentDemoTest {
+@TestPath("vaadin-form-layout")
+public class FormLayoutIT extends AbstractComponentIT {
+
+    private TestBenchTestCase layout;
 
     @Before
     public void init() {
+        open();
         Assert.assertTrue(isElementPresent(By.tagName("vaadin-form-layout")));
+        layout = this;
```

Comment:
I just realized that this can probably be avoided by removing the references to `layout` elsewhere in the code and simply call `findElement` instead.

Reply:
Sure, that's an option as well. I picked up this style from trying to minimize changes in https://github.com/vaadin/flow-components/pull/2774 (24 `layout` references in `ButtonIT`). Maybe we can prefer removing the `layout.` from now on.

Reply:
Done